docs/examples/grpc_server.md

```diff
@@ -75,11 +75,11 @@ If you already have an ACM certificate (including wildcard certificates) for the
 
 1. Wait a few minutes for the ALB to provision and for DNS to update.
 
-1.  Check the logs for `external-dns` and `aws-load-balancer-controller` to ensure the ALB is created and external-dns creates the record and points your domain to the ALB.
+1.  Check the `aws-load-balancer-controller` logs to ensure the ALB is created. Also ensure that `external-dns` creates a DNS record that points your domain to the ALB.
 
     ```bash
-    kubectl logs -n kube-system $(kubectl get po -n kube-system | egrep -o 'aws-load-balancer-controller[a-zA-Z0-9-]+') | grep 'grpcserver\/grpcserver'
```

Comment:
The old command does not work because the `k get po` command returns 2 pods. So the `k logs` command fails with - 
```
error: container <pod-1> is not valid for pod <pod-2>
```
I am guessing it worked before because there was only 1 replica running.